source/brailleDisplayDrivers/baum.py

```diff
@@ -162,11 +188,17 @@ def _onReceive(self, data):
 		self._handleResponse(command, arg)
 
 	def _handleResponse(self, command, arg):
+		"""
+		:type command: bytes
+		:type arg: bytes
+		"""
 		if command == BAUM_CELL_COUNT:
+			# Assumption: BAUM_CELL_COUNT command has a single byte unsigned argument.

```

Comment:
A cell count usually is not much more than 80.

Reply:
Within the scope of braille displays, a cell count is pretty self explanatory. I think you can leave the comment as is.